lib/Controller/SignalingController.php

```diff
@@ -353,7 +353,7 @@ public function signaling(string $token, string $messages): DataResponse {
 	 * Get signaling messages
 	 *
 	 * @param string $token Token of the room
-	 * @return DataResponse<Http::STATUS_OK|Http::STATUS_NOT_FOUND|Http::STATUS_CONFLICT, array{type: string, data: TalkSignalingSession[]}[], array{}>|DataResponse<Http::STATUS_BAD_REQUEST, string, array{}>
+	 * @return DataResponse<Http::STATUS_OK|Http::STATUS_NOT_FOUND|Http::STATUS_CONFLICT, array{type: string, data: TalkSignalingSession[]|string}[], array{}>|DataResponse<Http::STATUS_BAD_REQUEST, string, array{}>
```

Comment:
I don't see data bring a string anywhere?

Reply:
Let's improve readability:

```suggestion
	 * @return DataResponse<Http::STATUS_OK|Http::STATUS_NOT_FOUND|Http::STATUS_CONFLICT, list<array{type: string, data: TalkSignalingSession[]|string}>, array{}>|DataResponse<Http::STATUS_BAD_REQUEST, string, array{}>
```

Reply:
It is when it comes from https://github.com/nextcloud/spreed/blob/cd20ba221b582ba95bdbad7e70221eb71569ddc5/lib/Controller/SignalingController.php#L392. I think it should just be an array as well, but it is not because it comes directly from the DB.

Reply:
right.